homeassistant/components/geniushub/sensor.py

```diff
@@ -58,6 +58,32 @@ def name(self):
         """Return the name of the sensor."""
         return self._name
 
+    @property
+    def icon(self):
+        """Return the icon of the sensor."""
+        child_values = self._device._info_raw['childValues']  # noqa; pylint: disable=protected-access
+
+        last_comms = datetime.fromtimestamp(child_values['lastComms']['val'])
+        interval = child_values['WakeUp_Interval']['val']
+
+        if last_comms < datetime.now() - timedelta(seconds=interval * 3):
```

Comment:
Use utcnow, not now.

Reply:
Done. Is it because you don't want naive dt objects?